tests/aslts/bin/Do

```diff
@@ -467,15 +484,18 @@ run_asl_compiler()
 		list=`echo "$3" | cut -c 3-`
 	fi
 
+	list=`echo $list | rev | cut -c 4- | rev ` # cut the last item from the list. It's a yes or no from the diasm option.
```

Comment:
Too long comments, you may make it a separate line.
And what does the diasm mean?